usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -577,3 +585,91 @@ def fetch_naics_hierarchy(naics: str) -> dict:
         pass
     results = {"toptier_code": toptier_code, "midtier_code": midtier_code, "base_code": base_code}
     return results
+
+
+defc_sql = """
```

Comment:
I recommend putting this and any functions in this .py file that use this or build upon this into a different module named `usaspending_api/awards/v2/data_layer/sql.py`. Being in `orm.py` implies that I could find data access code that uses the Django ORM. This goes straight to SQL so isn't really orm-code.